classes/class-p4-master-site.php

```diff
@@ -547,12 +547,14 @@ public function enqueue_public_assets() {
 		wp_enqueue_style( 'slick', 'https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.9.0/slick.min.css', [], '1.9.0' );
 
 		// This loads a linked style file since the relative images paths are outside the build directory.
-		wp_enqueue_style( 'parent-style', $this->theme_dir . '/assets/build/style.min.css', [], $css_creation );
+		wp_enqueue_style( 'parent-style', $this->theme_dir . '/assets/build/style.min.css?' . time(), [], $css_creation );
```

Comment:
Do we needed the time value appended every time to css file? I mean the $css_creation already add as a updated version number if file got updated. 🤔

Reply:
Yes, this is vestigial! I was just debugging because I was having caching issues.

(Sorry for the late response, this was de-prioritized, now it came back to life!)